README.md

```diff
@@ -171,6 +171,19 @@ interval between any two periodic sync events can be greater than the requested
 is up to the browser implemetation. Also, the interval between any two periodic sync events for a
 registration doesn't have to be the same.
 
+Since a website will most likely download resources from the network when processing a
+`periodicsync` event, multiple `periodicsync` events can cause a lot of data consumption.
+Like other service worker events, `periodicsync` is subject to the service worker execution timeout.
```

Comment:
Most specs say that implementations _MAY_ have timeouts for these events, so this isn't a valid assertion to make.

Reply:
updated to include a 'MAY'.